_connect-files/api/objects/form-properties/sources/databases/mysql-object.md

```diff
@@ -32,4 +32,48 @@ description: ""
 
 uses-common-fields: true
 uses-feature-fields: true
+
+object-attributes:
+  - name: "ssl_ca"
+    type: "string"
+    required: false
+    description: |
+      **Optional**: The certificate (typically a CA or server certificate) Stitch should verify the SSL connection against. The connection will succeed only if the server's certifcate verifies against the certificate provided.
+
+      **Note**: Providing a certifcate via this property isn't required to use SSL. This is only if Stitch should verify the connection against a specific certificate.
+    value: "<SSL_CERTIFICATE>"
+  
+  - name: "ssl_cert"
+    type: "string"
+    required: false
+    description: |
+      **Optional**: If `ssl_client_auth_enabled: true`, the SSL client authentication cerficiate stitch should use. The `ssl_key` property must also be provided to ensure the connection is successful.
+    value: "<CA_CERTIFICATE>"
+
+  - name: "ssl_client_auth_enabled"
+    type: "string"
+    required: false
+    description: |
+      **Optional**: Indicates if SSL client authentication should be used. Accepted values are:
+
+      - `true`
+      - `false`
+    value: "true"
+
+  - name: "ssl_key"
+    type: "string"
+    required: false
+    description: |
+      **Optional**: If `ssl_client_auth_enabled: true`, the SSL client authentication key stitch should use. The `ssl_ca` property must also be provided to ensure the connection is successful.
+    value: "<CA_KEY>"
+    
+  - name: "verify_mode"
```

Comment:
There should also be a `check_hostname` property here. So, where `verify_mode` validates the certificate signing chain, `check_hostname` is another parameter that works with it.

If enabled, it will validate that the hostname of the database server matches the names in the certificate. If someone wants to not use a custom CA, they should enable both of those.

Reply:
Awesome - I'll add this in and roll it out :+1: